charts/redpanda/values.go

```diff
@@ -241,7 +263,7 @@ type Statefulset struct {
 	// Annotations are used only for `Statefulset.spec.template.metadata.annotations`. The StatefulSet does not have
 	// any dedicated annotation.
 	Annotations map[string]string `json:"annotations" jsonschema:"deprecated"`
-	PodTemplate PodTemplate       `json:"podTemplate"`
+	PodTemplate PodTemplate       `json:"podTemplate" jsonschema:"required"`
```

Comment:
I don't understand why every field under `PodTemplate` including `PodTemplate` is now required field?

Reply:
The JSONSchema applies to the result of merging the users values and our values. By specifying required, we're really just ensuring that they can not be set to `null`.